<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentdashboard</class>
 <widget class="QDialog" name="studentdashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 250)</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>21</y>
     <width>141</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 128, 128); 
    color: white;
    border: none;
    padding: 10px 20px;
    font-size: 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: rgb(32, 178, 170); 
}

QPushButton:focus {
    background-color: rgb(32,178,170); 
    border: 2px solid rgb(0, 128, 128); 

}
</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="assignmentButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 128, 128); 
    color: white;
    border: none;
    font-size: 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: rgb(32, 178, 170); 
}

QPushButton:focus {
    background-color: rgb(32,178,170); 
    border: 2px solid rgb(0, 128, 128); 

}
</string>
      </property>
      <property name="text">
       <string>ASSIGNMENT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="internalButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 128, 128); 
    color: white;
    border: none;
    padding: 10px 20px;
    font-size: 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: rgb(32, 178, 170); 
}

QPushButton:focus {
    background-color: rgb(32,178,170); 
    border: 2px solid rgb(0, 128, 128); 

}
</string>
      </property>
      <property name="text">
       <string>INTERNAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logOutButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 128, 128); 
    color: white;
    border: none;
    padding: 10px 20px;
    font-size: 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: rgb(32, 178, 170); 
}

QPushButton:focus {
    background-color: rgb(32,178,170); 
    border: 2px solid rgb(0, 128, 128); 

}
</string>
      </property>
      <property name="text">
       <string>LOG OUT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>941</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(194, 203, 201); </string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Home">
    <widget class="QCalendarWidget" name="calendar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>941</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 128, 128);

calendarWidget-&gt;setStyleSheet(
    &quot;QCalendarWidget QAbstractItemView:enabled:selected {&quot;
    &quot;    background-color: rgb(100, 149, 237); /* Cornflower Blue */&quot;
    &quot;    color: white;&quot;
    &quot;}&quot;
    &quot;QCalendarWidget QAbstractItemView:enabled:hover {&quot;
    &quot;    background-color: rgb(173, 216, 230); /* Light Blue */&quot;
    &quot;}&quot;
    &quot;QCalendarWidget QAbstractItemView[date=\&quot;2024-07-17\&quot;]:enabled {&quot;
    &quot;    background-color: rgb(255, 192, 203); /* Pink */&quot;
    &quot;}&quot;
);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-color:black;</string>
     </property>
     <property name="text">
      <string>Internal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pictures.qrc">:/images/images/redColour.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>0</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pictures.qrc">:/images/images/yellowColour.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-color:black;</string>
     </property>
     <property name="text">
      <string>Assignment</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Assignment">
    <widget class="QTableView" name="AssinmentTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>531</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Internal">
    <widget class="QTableView" name="internalTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
